### ISSUE TITLE
How to add self signed certificate permanently?

### ISSUE THREAD

**Issue by Haxy89** [brave/brave-browser#1268]
Every time I open server in my local network with self signed certificate I must accept it in Brave browser.
How to add exception permanently like in Chrome or Firefox?

**Comment by filipemir** [comment]
Is there no plan to add this as a feature? It's helpful for local development.

**Comment by bsclifton** [comment]
@filipemir you should be able to set this:
brave://flags/#allow-insecure-localhost

Give that a shot (set to `Enabled`) and let us know if it works 😄

**Comment by filipemir** [comment]
Thanks @bsclifton. It does appear to be working but I'm not sure whether it was related to that flag. I have a hosts override that directs local.my-companys-site.com to localhost (for silly legacy reasons that's needed for our site to work properly locally). Before I added my question I was having to accept the self-signed certificate each time I visited the page. But that stopped happening at some point before I set the flag you mentioned to enabled. Not sure why. I have now enabled the flag and I will let you know if I run into issues either way.
